corehq/apps/export/models/new.py

```diff
@@ -1617,6 +1620,51 @@ def _process_app_build(cls, current_schema, app, form_xmlns):
         return cls._merge_schemas(*schemas)
 
     @classmethod
+    def _add_export_items_from_subcase_action(cls, group_schema, root_path, subcase_action, repeats):
+        """return (path, label) tuples for each export item"""
```

Comment:
I don't understand this docstring. This function doesn't appear to return anything.

Reply:
I'm about to push a whole set of changes to this